pkg/validate/validate.go

```diff
@@ -166,3 +166,14 @@ func Files(fileNames []string, validationRules []*validationrule.ValidationRule,
 	validationReport.Duration = time.Since(start)
 	return validationReport
 }
+
+func generateExcludedRules(excludedRulesText string) []string {
```

Comment:
Another thing that comes up to my mind is that such a function might also deduplicate the rule names :thinking: 

Not insisting on it but would make sense

Reply:
added